pkg/kubelet/kubelet_network_linux.go

```diff
@@ -37,8 +37,12 @@ const (
 )
 
 func (kl *Kubelet) initNetworkUtil() {
-	iptClients := utiliptables.NewDualStack()
-	if len(iptClients) == 0 {
+	iptClients, err := utiliptables.NewDualStack()
+	if err != nil {
+		klog.ErrorS(err, "Failed to initialize iptables")
+	}
+
+	if err != nil || len(iptClients) == 0 {
```

Comment:
This also does not look correct, right @danwinship and @aroradaman ? despite err being not nil there can still be a single ip family present that we'll use

Reply:
hey, thanks for responding, no problem at all!